### ISSUE TITLE
[Task]: Make the datepicker in property pane keyboard accessible

### ISSUE THREAD

**Issue by aswathkk** [appsmithorg/appsmith#10095]
Insert a date picker widget and look at the property pane to see the date picker.

![Screenshot 2021-12-31 at 8 27 40 PM](https://user-images.githubusercontent.com/10436935/147829404-c523d9c8-8004-484c-8e8e-e02afd8938c7.png)

Reference: [HTML input type "datetime-local"](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_datetime-local)

**Comment by aswathkk** [comment]
As of now, I think the components are keyboard accessible already. We are not noticing it due to the lack of focus styles.
